src/content/about.md

```diff
@@ -0,0 +1,255 @@
+vendor  prefixes (like `-ms` or `-webkit`) to attributes, which is then in turn passed into the `merge()` plugin.
+
+The `merge()` plugin will copy the contents of each of its inputs into its output directory. Thus, it merges our 
+uglified JavaScript and out vendor prefixed css. This then becomes our final output and is what is written to 
+our target (destination) directory.
+
+This should all be fairly familiar to you if you've ever written JavaScript (or any programming language for that
+matter) before, it's just inputs and output.
+
+### Plugins
+
+Plugins are what a build pipeline developer will interact with most. Plugins are what do the actual work of
+transforming files at each step of build process. The API of a plugin is pretty simple, all that's required is
+creating a class that extends the [broccoli-plugin](https://github.com/broccolijs/broccoli-plugin) base class, and
+implementing a `build()` method, that performs some work or returns a promise.
+
+```js
+const Plugin = require('broccoli-plugin');
+
+class MyPlugin extends Plugin
+{
+  build() {
+    // A plugin can receive single or multiple inputs nodes/directories
+    // The files are available in this.inputPaths[0]/this.inputPaths[1]...
+    // You can do anything with these files that you can do in Node
+    // The output of your plugin must write to this.outputPath
+  }
+}
+```
+
+A broccoli-plugin has only one purpose, to transform the files from its `this.inputPaths` directories to its
+`this.outputPath` directory when its `build()` function is invoked. Anything you can do in Node, you can do in
+the `build()` method. A plugin can receive one or multiple inputs, and these are available in the `this.inputPaths`
+array in the order they are provided.
+
+`this.inputPaths` contains paths to directories, that are the `outputPath` of previous plugins. Each `inputPath`
+contains files that you can manipulate and write to `this.outputPath`. Broccoli will handle the state of these
+directories and take responsibility for passing them between plugins.
+
+There is a special case where a `string` is passed as an input to a plugin. When parsing your build pipeline, 
+Broccoli will automatically convert a string input into a
+[source plugin](https://github.com/broccolijs/broccoli-source). This plugin basically connects its input directory
+to its output directory, and also allows Broccoli to `watch` and be notified when files within the input
+directory change and trigger a rebuild. You can also manually create an `unwatched` directory from a string by
+using [UnwatchedDir](https://github.com/broccolijs/broccoli-source#new-unwatcheddirdirectorypath-options), 
+changes in this directory will not trigger a rebuild.
+
+### Trees
+
+The build pipeline is series of connected plugins, from one or more input directories to a single target directory.
+Working from the target directory (the final output) back up to the source directories resembles a tree. Think of a
```

Comment:
What you have here may be fine, but let me go into more detail so you can make the call if the docs are still appropriate or not.

---

The input structure isn't always tree, for example:

```js
const a = new Foo('foo');
const b = new Bar(a);

merge([a,b]);
```

Although the output of ever plugin is tree like.
Also internally broccoli does cycle breaking, and only processes the plugins as if they were a tree, but connects them as a directed graph.

Reply:
Sure, but I think from a mental model, it's still useful to think of it as a tree.
```
  a
/  \
|   b
\  /
 c
```
The above is more of a shrub than a tree

![](https://www.naturehills.com/media/wysiwyg/blog/Privacy_Fence_LG.jpg)

If you feel strongly about this, please say so, but I think the overall mental model is useful to think of like a tree.

Reply:
Nah, I'm onboard. I just wanted to make sure this detail doesn't detract from the mental model. I do agree with you, I think it's fine. Saying "broccoli trees are actually directed graphs" likely just confuses the story, for little gain.